src/options/use-class-literals.ts

```diff
@@ -0,0 +1,12 @@
+import { BooleanSupportOption } from 'prettier';
+import { CATEGORY_PUG } from '.';
+
+export const PUG_USE_CLASS_LITERALS: BooleanSupportOption = {
+	since: '1.10.0',
```

Comment:
`1.10.0` was already released and I will release `1.10.1` today as a bugfix version
This feature will potentially land in `1.11.0` (if you manage to handle this PR :wink:)

Reply:
I leave the release version because I don't know when it will be released :P